src/virt-install

```diff
@@ -80,6 +79,17 @@ ks_tmp.write("""
 touch /boot/coreos-firstboot
 %end
 """)
+if args.ostree_repo:
+    assert args.ostree_stateroot
+    assert args.ostree_remote
+    assert args.ostree_ref
+    ks_tmp.write(f"""
```

Comment:
`ks_tmp.write(f` I don't think the `f` should be in there?

Reply:
That's how we format the string right?

Reply:
(TBH I didn't know it existed until I saw it in one of jlebon's PRs somewhere)

Reply:
```
$ python3
Python 3.6.6 (default, Jul 19 2018, 14:25:17)
[GCC 8.1.1 20180712 (Red Hat 8.1.1-5)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> foo=42
>>> "{foo}"
'{foo}'
>>> f"{foo}"
'42'
>>>
```

Reply:
wow.. that's a new feature. https://realpython.com/python-f-strings/

haven't decided if I like it yet or not.. but this obviously is not a typo like I thought it was so: :+1:

Reply:
> haven't decided if I like it yet or not

You will. It's awesome!